sdk/core/azure-core-management/src/main/java/module-info.java

```diff
@@ -9,9 +9,14 @@
     exports com.azure.core.management.serializer;
     exports com.azure.core.management.exception;
 
-    opens com.azure.core.management to com.fasterxml.jackson.databind;
-    opens com.azure.core.management.exception to com.fasterxml.jackson.databind;
-    opens com.azure.core.management.implementation.polling to com.fasterxml.jackson.databind;
+    opens com.azure.core.management to
+        com.fasterxml.jackson.databind,
+        com.azure.core;
+    opens com.azure.core.management.exception to
```

Comment:
which class in `com.azure.core.management.exception` requires this change?

Reply:
On core-mgmt level, I am not able to predict which of subclass of ManagementError would have JsonFlatten. It is out of control of core-mgmt lib. But if anyone uses a subclass of it with JsonFlatten, you now need open to azure-core.